src/html/index.html

```diff
@@ -48,6 +48,7 @@
                         <div class="form-check"><input class="form-check-input" type="checkbox" value="" id="gravity" /> <label class="form-check-label" for="gravity">Gravity</label></div>
                     </form>
                 </div>
+                <div id="grid"></div>
```

Comment:
Cool. I think the other option is writing over the canvas directly. THis might a be moire tricky to make it stable, but I think is a better move over all